packages/dev/sharedUiComponents/src/fluent/hoc/syncedSliderLine.tsx

```diff
@@ -1,11 +1,19 @@
-import { IPropertyLineProps, PropertyLine } from "./propertyLine";
-import { SyncedSliderInput, SyncedSliderProps } from "../primitives/syncedSlider";
+import { type PropertyLineProps, PropertyLine, SplitPropertyLineProps } from "./propertyLine";
+import { type SyncedSliderProps, SyncedSliderInput } from "../primitives/syncedSlider";
 
-type KeyOf<O> = string & keyof O;
-export type SyncedSliderLineProps<O, K extends KeyOf<O>> = Omit<IPropertyLineProps, "children"> &
+export type SyncedSliderLineProps<O, K> = PropertyLineProps &
     Omit<SyncedSliderProps, "value" | "onChange"> & {
+        /**
+         * String key
+         */
         validKey: K;
+        /**
+         * Object where O[K] is a number
+         */
         obj: O;

```

Comment:
A lot of webapi (like Object and Reflect) call these `propertyKey` and `target`. I would use those names for improved consistency and clarity.
```suggestion
        /**
         * Property key
         */
        propertyKey: K;
        /**
         * Object where O[K] is a number
         */
        target: O;
```